src/less/tokens/tokens.less

```diff
@@ -0,0 +1,8 @@
+// DEFAULT CORE TOKENS
+@import "core-default.less";
+
+// OPTIONAL CORE TOKEN OVERRIDES (SKIPPED IF FILE DOES NOT EXIST)
+@import (optional) "core-@{ds}.less";
```

Comment:
A @ds global-var is always being passed in to the lessc compiler. The value is going to be "ds4" or "ds6". In the case of "ds6" it will look for a "core-ds6.less" file in the folder, and because it doesn't exist, it is essentially a no-op. For "ds4", the file does exists, therefore gets imported and the default values contained in "core-default.less" are overridden.

To be honest, I'm not sure this is the last step in our new direction, but we are getting iteratively closer I feel.